src/_guides/language/language-tour.md

```diff
@@ -3053,10 +3053,26 @@ class A {
 }
 {% endprettify %}
 
+<aside class="alert alert-info" markdown="1">
+  **[Dart 2](/dart-2.0) difference**:
+  In Dart 2, you **can't invoke** an unimplemented method **unless**
+  the receiver is of type `dynamic` and
+  any of its **supertypes define that method** (even if abstractly).
+  For more information, see the
+  [informal feature specification.](https://github.com/dart-lang/sdk/blob/master/docs/language/informal/nosuchmethod-forwarding.md)
+</aside>
```

Comment:
I don't think that this is accurate and/or necessary. Let's see what feedback we get from my comments in https://github.com/dart-lang/site-www/issues/442#issuecomment-357721649.

Reply:
How do you interpret the feedback you got? As far as I can tell, this note is still necessary (though it could be tweaked).

Reply:
We need an adjustment here: '(1) the receiver has static type `dynamic`; or (2) the dynamic type of the receiver has an implementation of `noSuchMethod` which is different from the one in class `Object`, and the static type of the receiver has that method (declared as abstract or with a body, both will suffice).'

I think we should mention the implementation of `noSuchMethod` here, because otherwise we'd expect the readers to discover on their own that the dynamic type of the receiver would have had a compile-time error _unless_ it has such a declaration of `noSuchMethod`, and I'm not sure they'll think of that automatically.

Reply:
Oops: If that method is declared with a body in one of the superclasses then we will _not_ "call an unimplemented method" after all, because it is implemented!

So we might want to say '(2) the dynamic type of the receiver has an implementation of `noSuchMethod` which is different from the one in class`Object`, the static type of the receiver has the method, and the receiver class does not have an implementation of the method.'

We may or may not wish to give an example (after all, this is just a short remark in the given context), but here's one: 

```dart
class A { foo() {}} 
class C implements A { noSuchMethod(i) => 42; }
main () => print(new C().foo() == 42); // 'true'
```